src/playbooks/with_build/install_sdwan_portal_existing_vm.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Please add an empty line at the end.

Reply:
Also, is the filename appropriate? Do we need a menu entry for this?

Reply:
1. Added empty line at the end
2. filename is ok
3. we don't need a menu option for this